atom/browser/api/atom_api_web_contents.cc

```diff
@@ -893,21 +939,13 @@ void WebContents::DidFinishNavigation(
 void WebContents::SecurityStyleChanged(
     content::SecurityStyle security_style,
     const content::SecurityStyleExplanations& explanations) {
-    std::string type = "unknown";
-    switch (security_style) {
-      case content::SECURITY_STYLE_UNAUTHENTICATED:
-      case content::SECURITY_STYLE_AUTHENTICATION_BROKEN:
-        type = "insecure";
-        break;
-      case content::SECURITY_STYLE_WARNING:
-        type = "warning";
-        break;
-      case content::SECURITY_STYLE_AUTHENTICATED:
-        type = "secure";
-        break;
-      default: break;
+    if (explanations.ran_insecure_content) {
+      Emit("security-style-changed", "active-mixed-content");
```

Comment:
actually forget about the two bools, let's just convert the `SecurityStyleExplanations` to a javascript object with whatever fields we want to hold everything